dbms/src/main/java/ch/unibas/dmi/dbis/polyphenydb/PolyphenyDb.java

```diff
@@ -156,20 +158,28 @@ public void join( final long millis ) throws InterruptedException {
         final Authenticator authenticator = new AuthenticatorImpl();
         final JdbcInterface jdbcInterface = new JdbcInterface( transactionManager, authenticator );
         final HttpServer httpServer = new HttpServer( transactionManager, authenticator, RuntimeConfig.WEBUI_SERVER_PORT.getInteger() );
+        // prolly overkill just for testing
+        final StatisticQueryProcessor statisticQueryProcessor = new StatisticQueryProcessor( transactionManager, authenticator );
```

Comment:
Is this required for the actual module or only for testing? Can it be removed? Also in what way is it "overkill"?